pkg/apis/policy/v1alpha1/propagation_types.go

```diff
@@ -443,6 +452,30 @@ const (
 	DynamicWeightByAvailableReplicas DynamicWeightFactor = "AvailableReplicas"
 )
 
+// PreemptionBehavior describes whether and how to preempt resources that are
+// claimed by lower-priority PropagationPolicy(ClusterPropagationPolicy).
+// +enum
+type PreemptionBehavior string
+
+const (
+	// PreemptAlways means that preemption is allowed.
+	//
+	// If it is applied to a PropagationPolicy, it can preempt any resource as
+	// per Priority, regardless of whether it has been claimed by a PropagationPolicy
+	// or a ClusterPropagationPolicy, as long as it can match the rules defined
+	// in ResourceSelector. In addition, if a resource has already been claimed
+	// by a ClusterPropagationPolicy, the PropagationPolicy can still preempt it
+	// without considering Priority.
+	//
+	// If it is applied to a ClusterPropagationPolicy, it can only preempt from
+	// ClusterPropagationPolicy, and from PropagationPolicy is not allowed.
+	PreemptAlways PreemptionBehavior = "AlwaysPreempt"
```

Comment:
What about just `Always` and `Never`?

Reply:
+1, That makes sense. I can imagine how it looks in a yaml:
```
preemption: Always
```
or 
```
preemption: Never
```